webhook/mutate/webhook_test.go

```diff
@@ -0,0 +1,266 @@
+package mutate
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	v1 "k8s.io/api/admission/v1"
+)
+
+func TestMutatesValidRequest(t *testing.T) {
+	rawJSON := `{
+		"apiVersion": "admission.k8s.io/v1",
+		"kind": "AdmissionReview",
+		"request": {
+			"uid": "705ab4f5-6393-11e8-b7cc-42010a800002",
+			"namespace": "injectable",
+			"operation": "CREATE",
+			"object": {
+				"kind": "Pod",
+				"apiVersion": "v1",
+				"metadata": {
+					"name": "testpod",
+					"namespace": "injectable",
+					"creationTimestamp": null,
+					"labels": {
+						"name": "testpod",
+						"marblerun.marbletype": "test"
+					}
+				},
+				"spec": {
+					"containers": [
+						{
+							"name": "testpod",
+							"image": "test:image",
+							"command": [
+								"/bin/bash"
+							],
+							"terminationMessagePath": "/dev/termination-log",
+							"terminationMessagePolicy": "File",
+							"imagePullPolicy": "IfNotPresent"
+						}
+					]
+				},
+				"status": {}
+			},
+			"oldObject": null,
+			"dryRun": false,
+			"options": {
+				"kind": "CreateOptions",
+				"apiVersion": "meta.k8s.io/v1"
+			}
+		}
+	}`
+
+	CoordAddr = "coordinator-mesh-api.marblerun:25554"
+
+	// test if patch contains all desired values
+	response, err := mutate([]byte(rawJSON), true)
+	if err != nil {
```

Comment:
Maybe use `require` and `assert` to differentiate required setup steps and the actual asserted values.
E.g. 
```
require := require.New(t)
require.NoError(err, "failed to mutate request ")
```
and
```
assert := assert.New(t)
assert.Contains(string(r.Response.Patch), `{"op":"add","path":"/spec/tolerations","value":{"key":"kubernetes.azure.com/sgx_epc_mem_in_MiB"}}`, "patch contained sgx tolerations, but tolerations were not supposed to be set")
```